third_party/move/move-compiler-v2/tests/checking/inlining/lambda3.move

```diff
@@ -0,0 +1,99 @@
+module 0x8675309::M {
+    // use 0x1::XVector;
+
+    // public inline fun foreach<T>(v: &vector<T>, action: |&T|) { // expected to be not implemented
```

Comment:
I've added a comment at the top to explain what we're doing here, commenting out lines to suppress errors that were shown for lambda2.move, so that shadowed errors can be revealed while keeping line numbers identical to facilitate output matching with V1.